101-streamanalytics-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "streamAnalyticsJobName": {
+      "value": "StreamAnalyticsJobName"
+    },
+    "numberOfStreamingUnits": {
```

Comment:
Is it likely to change from one environment to the next (e.g. dev, test, prod)?  If so then a param makes sense.